doc/source/change_proposal_process.rst

```diff
@@ -110,37 +103,46 @@ These steps are necessary to initialize a project repo enabling you to contribut
     you are overhauling one section, adding new subsections to it and
     do a general proofreading fixing spelling typos, these should go
     into separate commits.
+    
+Preparing a pull request
+========================
 
 14. Now the commit (which is the Git terminology for “change set”) is
     in your local copy of your fork of the project. To upload it your
     fork on GitHub run:
 
     git push --set-upstream origin tutorial
 
-15. You may repeat steps 9 – 15 also arbitrary times Git answers with
-    a short report and an URL where you can create a pull
+
+15. Git answers with a short report and an URL where you can create a pull
     request. Copy it and open it in your browser:
 
     https://github.com/Nils-Magnus/docs_on_docs/pull/new/tutorial
 
+ 
 16. Enter a description in the GitHub UI for your reviewing colleagues
     and peers. Once finished, click on the green button “Create pull
     request”.
+    
+Varify build process and identify errors
+========================================
 
-17. The review phase starts. Now two things happen in parallel: The
+17. Now two things happen in parallel: The
     automated and the manual review. Let’s first look into the
-    automated checks: Zuul is instantly simulating to merge your
-    changes and build the whole project. If all is well, a green
+    automated checks: Zuul is simulating to merge your
+    changes and builds the whole project. If all is well, a green
     checkmark says: “This branch has no conflicts with the base
-    branch.” If, however, Zuul and Sphinx are not able to build the
+    branch.” In this case you can proceed to step 22.
+      
+18. If, however, Zuul and Sphinx are not able to build the
```

Comment:
In case of failures "... checks" is not the only way. You can directly go to the "checks" tab in the UI

Reply:
if you incist will add this extra way.

Reply:
you are right! This a very good point thanks for that idea